packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -682,6 +694,151 @@ class Viewport implements IViewport {
     return true;
   }
 
+  /**
+   * Resets the pan offset.  To do this, the options values
+   *    initialCenter and canvasCenter
+   * are applied to the vtk active camera, then the current pan
+   * value is read and is set as the initial conditions.  That
+   * sets current pan to 0,0
+   * The initialCenter is the IMAGE percentage coordinates for the
+   * image synchronization point.  That point is displayed at the
+   * percentage canvasCenter point.
+   * For example, suppose the imageCenter is (1,0.5) and the canvasCenter
+   * is (0.5,1), then on an image 513x513 on a 1025x801 canvas, the
+   * image point (256,512) will be displayed at canvas position (512,800)
+   * The initial pan values will then be 0,0.
```

Comment:
Again these should not be part of the function documentation, as it makes the doc harder to read. These are the definitions of the initialCenter and canvasCenter (which are not even arguments of this function)